src/move_basic.cpp

```diff
@@ -555,7 +509,6 @@ void MoveBasic::sendCmd(double angular, double linear)
 void MoveBasic::run()
 {
     ros::Rate r(20);
```

Comment:
20Hz is fine, but is there any specific reason for it being this instead of 15?

Reply:
20 > 15, therefore better 😛

Reply:
I will check the CPU, this is actually a really important parameter, because in some cases might also affect how fast we are getting the updated distance to the object - might change the collision avoidance.